doc/tutorial/text_analytics/working_with_text_data.rst

```diff
@@ -548,6 +548,33 @@ English.
 Bonus point if the utility is able to give a confidence level for its
 predictions.
 
+Pitfalls
```

Comment:
Does this really belong in this specific tutorial? It makes sense to link to from here, or to include its table of contents, but really this is just general usage advice.